README.md

```diff
@@ -1 +1,3 @@
-Hello World!
+# Arrow Flight Client Examples
+
+Dremio introduced the Arrow Flight server endpoint in the version 4.9.1 release. This repository shows Arrow Flight client examples for a selection of laguages supported in Arrow Flight. These examples are intended to help you get started on using Dremio with Arrow Flight.
```

Comment:
Use present tense.

Reply:
I'm calling it 'Dremio Arrow Flight service' in the release notes.  I would prefer to consistently refer to this as a service, unless PM disagrees. @tfry-dremio?

"Dremio 4.9.1 introduces the Dremio Arrow Flight Service.  This repository provides sample Arrow Flight clients in   several languages to help you get started with the new service."

Reply:
I will change it to use 'Dremio Arrow Flight Service', I used 'service' in the other README as well.